saleor/webhook/__init__.py

```diff
@@ -68,4 +78,5 @@ class WebhookEventType:
         ORDER_FULFILLED: "order.manage_orders",
         CUSTOMER_CREATED: "account.manage_users",
         PRODUCT_CREATED: "product.manage_products",
+        CHECKOUT_QUANTITY_CHANGED: "order.manage_orders",
```

Comment:
`"order.manage_orders"`?

Reply:
We migrated to enums to represent permissions, so it will be `OrderPermissions.MANAGE_ORDERS`.

Reply:
@maarcingebala Should we use order permissions for the checkout's actions?

Reply:
Normally checkout doesn't use any permissions, so maybe it's not required at all?

Reply:
@maarcingebala I've used the `order` permission as there isn't a one for the `checkout`. This webhook allows an extension to capture all the users' checkouts so it definitely requires some permission as we don't want everyone to be able to get this information.

I've picked `order` as I thought if you already can listen for the `orders` then `checkouts` don't have more information.

Reply:
I wouldn't mix these. Maybe simply need new permission for fetching checkouts?

Reply:
@maarcingebala I've added new `CheckoutPermission` and used it for this checkout's webhook. As for why we need it. We don't want every extension to be able to listen for the `checkout_quantity_changed` webhook as it exposes users  baskets. That should be explicit and possible only for dedicated extensions with proper permissions.